c2corg_api/views/book.py

```diff
@@ -42,14 +43,17 @@ def get(self):
 
     @restricted_json_view(
             schema=schema_create_book,
-            validators=[validate_book_create,
+            validators=[colander_body_validator,
+                        validate_id,
```

Comment:
I think this causes the failing unit tests. For `POST` you don't need `validate_id`.